### PR TITLE
Backers count

### PR DESCRIPTION
**Pull request by xdamman**
Original PR https://github.com/OpenCollective/website/pull/191

### CODE REVIEW

**Comment by xdamman** [comment]
This requires the api to return `group.yearlyIncome`, see https://github.com/OpenCollective/api/pull/306

File: frontend/src/components/public_group/PublicGroupHero.js [hunk 1]
```diff
@@ -1,7 +1,35 @@
 import React from 'react';
 import PublicTopBarV2 from '../../containers/PublicTopBarV2';
+import formatCurrency from '../../lib/format_currency';
+import filterCollection from '../../lib/filter_collection';
 
 export default class PublicGroupHero extends React.Component {
```
Comment (sedubois) [inline frontend/src/components/public_group/PublicGroupHero.js]: would be better to have this as functional component `export default ({group}) => {...}`

Comment (xdamman) [inline frontend/src/components/public_group/PublicGroupHero.js]: Why? What the difference?

Comment (sedubois) [inline frontend/src/components/public_group/PublicGroupHero.js]: It enforces not storing any state in React but rather in Redux, and will be subject to performance improvements in future react versions as the functional component can be flagged "pure". There is also probably less overhead to begin with, as there is no lifecycle management. But it's still an early feature. 

https://facebook.github.io/react/blog/2015/09/10/react-v0.14-rc1.html#stateless-function-components

> This pattern is designed to encourage the creation of these simple components that should comprise large portions of your apps. In the future, we’ll also be able to make performance optimizations specific to these components by avoiding unnecessary checks and memory allocations.

**Pull request merged**